schema.ts

```diff
@@ -723,24 +726,78 @@ export enum AttributionNodeType {
   Image,
 }
 
-interface TextAttributionNode {
+/**
+ * An attribution node that simply renders some text.
+ *
+ * This might be used to attribute the data source.
+ *
+ * @example
+ * ```
+ * coda.makeAttributionNode({
+ *   type: coda.AttributionNodeType.Text,
+ *   text: "Data provided by FooCorp.",
+ * });
+ * ```
+ */
+export interface TextAttributionNode {
+  /** Identifies this as a text attribution node. */
   type: AttributionNodeType.Text;
+  /** The text to render with the pack value. */
   text: string;
 }
 
-interface LinkAttributionNode {
+/**
+ * An attribution node that renders a hyperlink.
+ *
+ * This might be used to attribute the data source and link back to the home page
+ * of the data source or directly to the source data.
+ *
+ * @example
+ * ```
+ * coda.makeAttributionNode({
+ *   type: coda.AttributionNodeType.Link,
+ *   anchorUrl: "https://foocorp.com",
```

Comment:
Optional: I try to standardize on `example.com` for all sample URLs, because it's managed by IANA and guaranteed to be safe. Although unlikely, foocorp.com could one day have objectionable content hosted there.

Reply:
Aha, that's smart. Changed.